liborbital/bmp.rs

```diff
@@ -135,11 +136,6 @@ impl BmpFile {
         !self.data.is_empty()
     }
 
-    /// Convert to slice for drawing
-    pub fn as_slice(&self) -> &[Color] {
-        &self.data
```

Comment:
Well, first, what's the point of keeping `as_slice` since `&bmp` does the same thing and it's shorter?